src/blueapi/core/context.py

```diff
@@ -116,7 +139,7 @@ def with_dodal_module(self, module: ModuleType, **kwargs) -> None:
             )
             LOGGER.exception(NotConnected(exceptions))
 
-    def plan(self, plan: PlanGenerator) -> PlanGenerator:
+    def register_plan(self, plan: PlanGenerator) -> PlanGenerator:
```

Comment:
I don't object to this renaming, but will this or `register_device` break hyperion? @DominicOram @olliesilvester

Reply:
I don't think we use this in Hyperion yet, should be fine